lib/cryptoexchange/exchanges/dydx_perpetual/services/pairs.rb

```diff
@@ -17,6 +17,8 @@ def adapt(output)
             Cryptoexchange::Models::MarketPair.new(
               base:   base,
               target: target,
+              inst_id: value["symbol"],
```

Comment:
also need to use inst_id in ticker_url in market and orderbook
https://github.com/coingecko/cryptoexchange/blob/master/lib/cryptoexchange/exchanges/dydx_perpetual/services/order_book.rb#L17
https://github.com/coingecko/cryptoexchange/blob/master/lib/cryptoexchange/exchanges/dydx_perpetual/services/contract_stat.rb#L17

also need to add inst_id and contract_interval at L27 and L38 
https://github.com/coingecko/cryptoexchange/blob/master/lib/cryptoexchange/exchanges/dydx_perpetual/services/market.rb

Reply:
good catch! 👍